### DISCUSSION TITLE
Any specific reasson why you use private const val TAG instead of using Timber?

### DISCUSSION THREAD

**Discussion by mobilekosmos** [android/nowinandroid#843] [Q&A] [answered]
Using "Timber.d" avoids the need of a TAG const in each class, is there a reasson why this is till not part of the native Log class or why you are not using Timber?

**Comment by dturner** [comment, marked as answer]
The only reason we're not using Timber is that it's never become a priority to add it. Feel free to submit a PR.

**Reply by Moes81** [reply]
Is Timber still under development? The lastest version was released in Aug 2021. Would you still use Timber, even if it's no longer under maintenance?

**Reply by SimonMarquis** [reply]
> Is Timber still under development?

Probably not.

> Would you still use Timber, even if it's no longer under maintenance?

Generally, tiny libraries that are focused on doing one thing and doing it well do not need to be constantly updated. _If it aint broke dont fix it._

On the other hand, a logging library that would be constantly adding new feature should probably be avoided, for the exact same reason. Constant flow of updates probably means an endless source of potential new bugs :)
